### PR TITLE
update the readme and instructions

### PR DESCRIPTION
**Pull request by Mary-Snow**
## Context/Background

- updates readme for cleaner interview experience 

## Description of the Change

- after interviewing the candidate the readme could use some updates 
- updates the zeplin link so interviewees can access it 
- removes package not used

### CODE REVIEW

File: src/components/Pokemon/PokemonDetail/PokemonDetail.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import React from 'react';
  * {id: "1", Name: "Bulbasaur", Types: ["Grass", Poison], ..., Legendary: "False"}
  */
 
-const PokemonDetail = (props) => {
+const PokemonDetail = ({pokemon}) => {
```
Comment (pkmh) [inline src/components/Pokemon/PokemonDetail/PokemonDetail.jsx:9]: nice

File: README.md [hunk 2]
```diff
@@ -5,33 +5,20 @@
...
+   - HP, Attack, Defense, Sp. Atk, Sp. Def, and Speed should be centered in the card with the text left aligned
+4. Modify the search bar in order to search by name
 
 ### Bonus Tasks
 
-1. Have a search bar at the top that allows you to search by name
-2. Add some pagination (pageSize 10)
-3. Add a few tests
-4. How would you store media assets for this API and make the individual cards more dynamic?
-5. Add a filter endpoint for all the fire pokemon and water pokemon and displays the cards
```
Comment (pkmh) [inline README.md:24]: Why remove this? No biggie - just curious! :D

File: src/components/Pokemon/PokemonDetail/PokemonDetail.jsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ import React from 'react';
...
     return (
       <>
         <h1>{pokemon.Name}</h1>
```
Comment (pkmh) [inline src/components/Pokemon/PokemonDetail/PokemonDetail.jsx:12]: Should we also have a boiler-plate component for rendering the type correctly?

**Review by pkmh — COMMENTED** [review]
Looks great!! just had a couple small questions

File: src/components/Pokemon/PokemonDetail/PokemonDetail.jsx (hunk 3 above)
Comment (Mary-Snow) [reply src/components/Pokemon/PokemonDetail/PokemonDetail.jsx:12]: I was thinking that the candidate could decide how to render the types? Just to see how they go about and organize the logic?

File: README.md (hunk 2 above)
Comment (Mary-Snow) [reply README.md:24]: Candidates don't have time to get to this and we don't have any other routing requirements anymore. So it doesn't really fit in

Comment (pkmh) [reply README.md:24]: makes sense!

File: src/components/Pokemon/PokemonDetail/PokemonDetail.jsx (hunk 3 above)
Comment (pkmh) [reply src/components/Pokemon/PokemonDetail/PokemonDetail.jsx:12]: Makes sense too!

**Review by pkmh — APPROVED** [review]
Thanks for tackling this!! 🎉

**Pull request merged**